cli/src/cmds/clusters/add.rs

```diff
@@ -160,19 +160,19 @@ impl Command for AddCommand {
                     .default_value("2"),
             )
             .arg(
-                Arg::new("query_namespace")
-                    .long("query-namespace")
-                    .env(databend_query::configs::config_query::QUERY_NAMESPACE)
+                Arg::new("query_cluster_id")
+                    .long("query-cluster-id")
```

Comment:
how to get the numeric cluster id value on startup the query instance?  q.q

in etcd there's a `--initial-cluster-token` argument, which gives the cluster a human readable name
https://etcd.io/docs/v3.2/op-guide/configuration/#--initial-cluster-token

a human readable name is also valuable on setting the monitoring metrics, we can get some alerts about `cluster biz-etl01 loads too high` by the extracting the `cluster` tag inside the metric

Reply:
cluster-id is a string(uuid) style not numberic, but it can be set as "12345"

Reply:
```
                    .long("query-cluster-id")
```
oh sorry i missunderstood the `.long("")` method means the longer args name, not means `long` as args type, ignore me >.<